ARCHITECTURE.md

```diff
@@ -189,8 +189,8 @@ grandparent module. This restriction allows fast recalculation of the index:
 only the part corresponding to the changed files needs to be flushed.
 
 These indexes power go to class and go to symbol functionality. They are also
-used during resolve to find parent module for a file and to get the list of
-impls for a type.
+used during resolve to find the parent module for a file and to get the list of
```

Comment:
[Here](http://www.ibm.com/support/knowledgecenter/SSNW2F_5.2.1/com.ibm.p8.security.doc/p8psa074.htm)'s IBM talking about _**the** parent of a subfolder_.
[Here](https://docs.oracle.com/javase/7/docs/api/java/lang/ClassLoader.html)'s Oracle talking about _**the** parent of a `ClassLoader` instance_.
[Here](https://msdn.microsoft.com/en-us/windows/hardware/drivers/install/determining-the-parent-of-a-device)'s Microsoft talking about _**the** parent of a device_.